### PR TITLE
Swagger Correctness for Cognitive Services

### PR DESCRIPTION
**Pull request by yangyuan**
Hi all,

This is a update for fixing `Swagger Correctness` warnings from S360.
Mostly missing properties, enum values, should not be breaking changes.

### CODE REVIEW

**Comment by yangyuan** [comment]
@erich-wang looks like something is wrong with build, it says "no breaking change" but failed with system exception

**Comment by erich-wang** [comment]
rerun to check if the error is transient

**Comment by erich-wang** [comment]
@yangyuan , adding new optional properties is breaking change according to [Breaking Change Policy](http://aka.ms/bcforapi). you need to add new api version.

**Comment by yangyuan** [comment]
Hi @erich-wang, I was trying to explain this. We are not making any API change.
We are having this PR because of s360 alarms. We had these fields all the time but didn't include in API specs.

So this is more like a specs fix than a change.

If we create a new API version won't help as s360 alarms will still be there.

File: specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json [hunk 1]
```diff
@@ -985,6 +986,11 @@
           },
           "description": "Gets the capabilities of the cognitive services account. Each item indicates the capability of a specific feature. The values are read-only and for reference only."
         },
+        "dateCreated": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the date of cognitive services account creation."
+        },
```
Comment (erich-wang) [inline specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json:993]: Please move this property to end of its parent as it is broken change to some existing SDKs.

**Comment by yangyuan** [comment]
Hi @erich-wang, I have updated the PR and moved "dateCreated" to end of parent.

**Comment by erich-wang** [comment]
@yangyuan , although this PR is for S360 warning, it still needs approval based on [breaking change policy](https://aka.ms/AzBreakingChangesPolicy), I added label `BreakingChangeReviewRequired` to ask approval from review board. If you want this PR merged with urgency, please send email to @azbreakchangereview get approval sooner.

**Comment by yangyuan** [comment]
Hi @erich-wang, my understanding is that as long as you put label on this PR, review board will come and review right.
Is there any other action needed?

**Comment by erich-wang** [comment]
@yangyuan , please send email to azbreakchangereview to ask approval, thanks.

**Review by JeffreyRichter — APPROVED** [review]

**Pull request merged**